compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.cpp

```diff
@@ -180,6 +180,113 @@ LogicalObjectFifoFromMemrefOp DmaCpyNdOp::getTargetObjectFifo() {
   return dyn_cast<LogicalObjectFifoFromMemrefOp>(getTarget().getDefiningOp());
 };
 
+// Build a CircularDmaCpyNdOp with mixed static and dynamic entries.
```

Comment:
Does it have to be three build functions for static/dynamic entries?

Reply:
Yes, one for static, one for dynamic and one for static/dynamic mixed. I based myself on the `Subview` build functions for this: https://github.com/llvm/llvm-project/blob/2996cb7970fb536d7db1088d571dc8d2f9ad35fa/mlir/include/mlir/Dialect/MemRef/IR/MemRefOps.td#L1833 and https://github.com/llvm/llvm-project/blob/2996cb7970fb536d7db1088d571dc8d2f9ad35fa/mlir/lib/Dialect/MemRef/IR/MemRefOps.cpp#L2685